HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -24,6 +24,13 @@ def esproducers_by_type(process, *types):
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+def customiseFor14282(process):
```

Comment:
Because O2O is based on a new CondFormat, and L1T O2O is not yet ready, it seems some change to HLT configs is unavoidable.  The only thing missing is this EmptyESSource.